openwrt-openwisp-monitoring/files/monitoring.agent

```diff
@@ -213,6 +218,8 @@ send_data() {
 					sleep "$timeout"
 				fi
 			done
+			# compress file again
+			gzip "$filename"
```

Comment:
is it really needed?

Reply:
I think not now :thinking: 

I added it when we were checking for `.gz` files only. Removing it